.github/workflows/deploy-demo.yaml

```diff
@@ -0,0 +1,81 @@
+name: Deploy Frontend-Demo to GitHub Pages
+
+on: [workflow_dispatch]
+
+jobs:
+  check-package-lock:
```

Comment:
Should this file really include all checks (e.g. linting)? Or just do the deployment?